app/Http/Controllers/Api/ApiController.php

```diff
@@ -8,6 +8,11 @@
 
 abstract class ApiController extends Controller
 {
+    /**
+     * When true authorization is enabled
+     * @var bool
+     */
+    protected $should_authorize = false;
```

Comment:
Perhaps nit-picky, but should we use `snakeCase` for this property name? I don't believe that PSR-2 nor PSR-12 mandates this, but it's a style that we employ throughout the code-base, so it may be worth tweaking, if only for consistency. Sorry to be a pain!

Reply:
Thanks will update it.